bot/exts/filtering/_filter_lists/invite.py

```diff
@@ -18,6 +19,13 @@
     from bot.exts.filtering.filtering import Filtering
 
 
+REFINED_INVITE_CODE = re.compile(
+    r"(?P<invite>[a-zA-Z0-9/]+)"  # The supposedly real invite code.
```

Comment:
Would it be possible to add a comment showing what an example invite code looks like? Difficult to understand what a real invite code looks like

Reply:
It's really just a random combination of letters and numbers.